src/wallet/rpcdump.cpp

```diff
@@ -561,7 +561,11 @@ UniValue dumpwallet(const JSONRPCRequest& request)
             "dumpwallet \"filename\"\n"
             "\nDumps all wallet keys in a human-readable format.\n"
             "\nArguments:\n"
-            "1. \"filename\"    (string, required) The filename\n"
+            "1. \"filename\"    (string, required) The filename (if not full path, relative to bitcoind path)\n"
+            "\nResult:\n"
+            "{                           (json object)\n"
+            "  \"dumpfilepath\" : {        (string) The filename\n"
```

Comment:
I'd prefer to use the name "filename" (since this isn't just the path). You could change the help text to say "The filename with full absolute path"